<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Histogram_consulience_options</class>
 <widget class="QWidget" name="Histogram_consulience_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Канал А</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>№ канала</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sb_nudChANumber"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Поправка, нс</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="dsb_nudChACorr">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Цвета лучей</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QToolButton" name="btn_p"/>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="btn_gauss"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Gaus</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>P(∆t)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Метка времени</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Tint, нс</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="dsb_nudTInt"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Канал Б</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>№ канала</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Поправка, нс</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="sb_nudChBNumber"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="dsb_nudChBCorr">
            <property name="decimals">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Окно совпадений</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Tw, нс</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="dsb_nudWindow"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>N</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="dsb_nudN"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Исключать события, имеющие признаки:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item row="2" column="1">
         <widget class="QCheckBox" name="cb_PET_EVENT_FLAG_GATE_OVERLAY">
          <property name="text">
           <string>Признак наложения событий</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="cb_PET_EVENT_FLAG_SYNC">
          <property name="text">
           <string>Признак кардиосинхронизации</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="cb_PET_EVENT_FLAG_ADC_OVERFLOW">
          <property name="text">
           <string>Признак переполнения (потери значимости) данных АЦП</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="cb_PET_EVENT_FLAG_QDC_OVERFLOW">
          <property name="text">
           <string>Признак переполнения (потери значимости) кординатных сигналов</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="cb_PET_EVENT_FLAG_TS_INVALID">
          <property name="text">
           <string>Признак не корректности метки времени присвоенной событию</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="cb_PET_EVENT_FLAG_FAULT">
          <property name="text">
           <string>Признак сбоя работы устройства</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="cb_PET_EVENT_FLAG_QDC_UNDERFLOW">
          <property name="text">
           <string>Признак опустошения (потери значимости) кординатных сигналов</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="cb_PET_EVENT_FLAG_ADC_UNDERFLOW">
          <property name="text">
           <string>Признак опустошения (потери значимости) данных АЦП</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btn_accept">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sb_nudChANumber</tabstop>
  <tabstop>sb_nudChBNumber</tabstop>
  <tabstop>dsb_nudChACorr</tabstop>
  <tabstop>dsb_nudChBCorr</tabstop>
  <tabstop>dsb_nudTInt</tabstop>
  <tabstop>dsb_nudWindow</tabstop>
  <tabstop>dsb_nudN</tabstop>
  <tabstop>btn_p</tabstop>
  <tabstop>btn_gauss</tabstop>
  <tabstop>cb_PET_EVENT_FLAG_FAULT</tabstop>
  <tabstop>cb_PET_EVENT_FLAG_TS_INVALID</tabstop>
  <tabstop>cb_PET_EVENT_FLAG_GATE_OVERLAY</tabstop>
  <tabstop>cb_PET_EVENT_FLAG_QDC_OVERFLOW</tabstop>
  <tabstop>cb_PET_EVENT_FLAG_QDC_UNDERFLOW</tabstop>
  <tabstop>cb_PET_EVENT_FLAG_ADC_OVERFLOW</tabstop>
  <tabstop>cb_PET_EVENT_FLAG_ADC_UNDERFLOW</tabstop>
  <tabstop>cb_PET_EVENT_FLAG_SYNC</tabstop>
  <tabstop>btn_accept</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
